submissions/chernetskyi8704/Tiny JS World/index.js

```diff
@@ -0,0 +1,63 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: _put repo URL here_
+   Web app: _put project's github pages URL here_
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+class Inhabitant {
+  constructor(species, name, gender, saying) {
+    this.species = species;
+    this.name = name;
+    this.gender = gender;
+    this.saying = saying;
+  }
+
+  displayData() {
+    return ["species", "name", "gender", "saying"]
+      .map((data) => this[data])
```

Comment:
`data` is a generic term. What exactly `map` works with?

Reply:
I will take into account in the future, not to use a generic term in situations like that. Thanks a lot.